src/main/java/org/broadinstitute/hellbender/tools/gvs/common/PloidyUtils.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.tools.gvs.common;
+
+import htsjdk.samtools.util.CollectionUtil;
+import htsjdk.samtools.util.Interval;
+import htsjdk.samtools.util.Lazy;
+import htsjdk.variant.variantcontext.VariantContext;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+public class PloidyUtils {
+    /**
+     * This copies the PAR definition used in the tool FindMendelianViolations with the addition of the Y regions
+     * https://github.com/broadinstitute/picard/blob/master/src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java#L203
```

Comment:
you may want to use a permalink here in case the target changes out from under you, such as `https://github.com/broadinstitute/picard/blob/63138fc8b7ae33eb0490a8ef707b61befa2f51d4/src/main/java/picard/vcf/MendelianViolations/FindMendelianViolations.java#L203`

Reply:
Done.  Also updated it with a Javadoc @see link